### DISCUSSION TITLE
Add a possibility to clone alert

### DISCUSSION THREAD

**Discussion by gjemp** [grafana/grafana#48763] [Alerting]
It would be nice to be able to clone existing alert. It would simplify splitting alerts to smaller  alerts ( 10+ queries in one alert ) or use some existing query as "template" where some parameter  change is only needed for the new alert.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
